### PR TITLE
Fix Flyio GH action

### PR DESCRIPTION
**Pull request by aminalaee**
Fix Flyio Github action

### CODE REVIEW

File: .github/workflows/check-test-release.yml [hunk 1]
```diff
@@ -99,17 +99,18 @@ jobs:
...
+    - name: Run Flyio tests
+      if: matrix.os == 'ubuntu-latest' && matrix.python == '3.9'
+      run: pytest -k 'flyio'
     - name: Start minikube
       if: matrix.os == 'ubuntu-latest' && matrix.python == '3.9'
       uses: medyagh/setup-minikube@master
     - name: Run tests
       timeout-minutes: 40
-      run: pytest -k 'not heroku'
+      run: pytest -k 'not heroku and not flyctl'
```
Comment (aguschin) [inline .github/workflows/check-test-release.yml:113]: > https://github.com/iterative/mlem/blob/main/tests/contrib/test_heroku.py#L37-L39

I assume:
```suggestion
      run: pytest -k 'not heroku and not flyio'
```

Comment (aguschin) [inline .github/workflows/check-test-release.yml:113]: > FAILED tests/contrib/test_flyio.py::test_flyio_create_app - FileNotFoundError: [Errno 2] No such file or directory: 'flyctl'

I assume:
```suggestion
      run: pytest -k 'not heroku and not flyio'
```

**Comment by aminalaee** [comment]
You are right about `pytest -k 'not heroku and not flyctl'` but I think the dummy PR passed the pipeline. This PR is still using the old action, right?

**Comment by aguschin** [comment]
> I think the dummy PR passed the pipeline

Yes, looks like that!

> This PR is still using the old action, right?

Yes.

**Comment by aguschin** [comment]
Yes, CI finished. There was some error with catboost, I've seen it in earlier PRs. We'll need to fix it, if it doesn't take care of itself, in order to release next version of MLEM. Merging! Thanks for the quick fix @aminalaee !

**Review by aguschin — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**